README.md

```diff
@@ -7,3 +7,60 @@
 This is a continuous-time simulation with the simplest-possible state and non-exponential transitions. The goal is to make a compact, fastest-possible non-exponential continuous-time simulation.
 
 The simplest-possible state is a vector-addition system, which has a state that is a vector of integers just like chemical equations. A transition is defined by a vector of integers that represent how many values to take or give. However, for this simulation, the transition rates can be any function of the state and have exponential or non-exponential distributions.
+
+## Vector Addition System
+
+A [vector addition system](https://en.wikipedia.org/wiki/Vector_addition_system) (VAS) is a simple mathematical language to model systems. It is simpler than the Petri net language, but has the advantage of allowing for more free-form simulation design, as intensity functions can be arbitrary functions of state.
+
```

Comment:
This is a good start. The first paragraph is right for the README. Maybe the rest should be in the docs directory. It needs to be expanded, too. This is such a direct way to write a simulation, but I don't know people think of it often these days.